src/Uno.UWP/UI/ViewManagement/UISettings.cs

```diff
@@ -53,13 +65,21 @@ public Color GetColorValue(UIColorType desiredColor)
 			var systemTheme = SystemThemeHelper.SystemTheme;
 			return desiredColor switch
 			{
+
+#if __MACOS__
+				UIColorType.Background => NSColor.ControlBackground,
+				UIColorType.Foreground => NSColor.ControlText,
+				UIColorType.Accent => NSColor.ControlAccent,
+#else
 				UIColorType.Background =>
 					systemTheme == SystemTheme.Light ? Colors.White : Colors.Black,
 				UIColorType.Foreground =>
 					systemTheme == SystemTheme.Light ? Colors.Black : Colors.White,
 				// The accent color values match SystemResources.xaml in Uno.UI
 				// as we can't access Application resources from here directly.
 				UIColorType.Accent => Color.FromArgb(255, 51, 153, 255),
+#endif
+
 				UIColorType.AccentDark1 => Color.FromArgb(255, 0, 90, 158),
```

Comment:
We need to build those colors as well, I'll looking up the proper way to build those, I'll let you know.

Reply:
We're still gathering information on how to build those colors, I'll let you know when we have the correct inforation.